kivy_garden/graph/__init__.py

```diff
@@ -1527,6 +1527,37 @@ def plot_mesh(self, *args):
             vert[k * 8 + 5] = px_ymax
         mesh.vertices = vert
 
+class ScatterPlot(Plot):
+    """
+    ScatterPlot draws using a standard Point object.
+    The pointsize can be controlled by passing `pointsize` when initialising
+    the class. Default pointsize is 1.0.
+
+    >>> plot = ScatterPlot(color=[1, 0, 0, 1], pointsize=5)
+    """
+
+    pointsize = NumericProperty(1)
```

Comment:
This should be `point_size`.

Reply:
Yes, I'll change this. But in kivy's Point class it seems to be called `pointsize`.

Reply:
Yes, but that probably should also have been `point_size`.